hnn_core/network.py

```diff
@@ -274,7 +274,7 @@ def _instantiate_feeds(self, n_trials=1):
         cur_params = self.params.copy()  # these get mangled below!
         for trial_idx in range(n_trials):
 
-            prng_seedcore_initial = cur_params['prng_*'].copy()
```

Comment:
move it outside the loop?

Reply:
> move it outside the loop?

It needs to be inside the loop so that `prng_seedcore_initial` is reset to the original value for each trial, then modified.

Reply:
Hum, yeah, I guess I still struggle understanding how seeds are being messed around with. @jasmainak is right I think: L277 could (and therefore should?) come out of the trial_idx-loop. It's `cur_params` being modified, not `prn_seedcore_initial`.

Reply:
Ohhh, I see it now.

Reply:
I don't know if this is the right time to make this fix, but what do you think of the following:

- first try to add an argument called `n_trials` to [these functions](https://github.com/jonescompneurolab/hnn-core/blob/master/hnn_core/feed.py#L191). Then do the for loop inside this function that updates the `prng` for each trial. These functions will then return a list of list instead of list.

- We can keep that behavior for a while to gain people's trust on hnn-core outputs

- Next, we realize the fact that incrementing the `prng` in a for loop is equivalent to not incrementing the `prng` at all. That's because the random number generator's state has moved after creating say 100 random numbers, so there is no need to update `prng`. And thus, we get rid of the bad seed updating behavior for good.

Wdyt?

Reply:
I like that a lot actually. Let me play around with this a little and we can rebase out any commits we don't actually want to run with.